src/Handler/StreamHandler.php

```diff
@@ -97,31 +100,40 @@ public function withEncoder(callable $encoder): HandlerInterface
 
     private function initializeStream(): void
     {
-        if (is_string($this->uri)) {
-            if (
-                str_starts_with($this->uri, 'udp://') ||
-                str_starts_with($this->uri, 'udg://') ||
-                str_starts_with($this->uri, 'tcp://') ||
-                str_starts_with($this->uri, 'unix://')
-            ) {
-                $this->stream = fsockopen($this->uri);
-            } else {
-                $this->stream = fopen($this->uri, 'wb+');
-            }
+        if (!is_string($this->uri)) {
```

Comment:
Isn't it better not to use two more variables and set already allocated variable for "stream" that's anyway has `mixed` type?

Reply:
Fixed.